packages/sync/src/class-replicastore.php

```diff
@@ -867,9 +867,13 @@ public function delete_site_option( $option ) {
 	 * @access public
 	 *
 	 * @param string $taxonomy Taxonomy slug.
-	 * @return array Array of terms.
+	 * @return array|\WP_Error Array of terms or WP_Error object on failure.
```

Comment:
Or false.

What's the difference in semantics between WP_Error and false, or false and an empty array?

Reply:
Never mind, I was confused. There is no false, when `ensure_taxonomy` declared it can return bool it was referring to true.